pkg/util/metric/m_fileservice.go

```diff
@@ -0,0 +1,29 @@
+package metric
+
+var FsMemCacheReadCounter = NewCounterVec(
+	CounterOpts{
+		Subsystem: "fs",
+		Name:      "mem_cache_read_total",
+		Help:      "Counter for Memcache Reads",
+	},
+	[]string{constTenantKey, "type"},
+	false,
+)
+
+var FsS3ReadCounter = NewCounterVec(
+	CounterOpts{
+		Subsystem: "fs",
+		Name:      "s3_read_total",
+		Help:      "Counter for S3 Reads",
+	},
+	[]string{constTenantKey, "type"},
+	false,
+)
+
+func MemCacheReadCounter() Counter {
+	return FsMemCacheReadCounter.WithLabelValues()
```

Comment:
WithLabelValues() is empty.

Reply:
if the `type` label is not required, you don't have to use `WithLabelValues`, just:
```go
var FsS3ReadCounter = NewCounter(CounterOpts{
	Subsystem: "fs",
	Name:      "s3_read_total",
	Help:      "Counter for S3 Reads",
	ConstLabels: sysTenantID,
})
```
and call FsReadCounter.Inc() directly in fileservice package

Reply:
Thank you. I will update it.